generator/src/main/java/io/wcm/devops/conga/generator/Generator.java

```diff
@@ -64,6 +64,10 @@ public Generator(GeneratorOptions options) {
    * @param environmentNames Environments to generate. If none specified all environments are generated.
    */
   public void generate(String... environmentNames) {
+    generate(environmentNames, new String[] {});
+  }
+
+  public void generate(String[] environmentNames, String... nodeNames) {
```

Comment:
i would rather use simply array syntax [] for all input paraemters - mixture of a string array and a varargs array is more puzzling than it helps. probably also for the other generate method and the generate method in the EnvironmentGenerator.